client/src/main/java/io/pravega/client/batch/impl/BatchClientImpl.java

```diff
@@ -65,7 +65,12 @@ public BatchClientImpl(Controller controller, ConnectionFactory connectionFactor
         latestDelegationToken = new AtomicReference<>();
     }
 
+    /**
+     * Used to fetch the StreamInfo of a given stream. This should be removed in time.
+     * @deprecated Use {@link io.pravega.client.admin.StreamManager#getStreamInfo(String, String)} to fetch StreamInfo.
+     */
     @Override
+    @Deprecated
```

Comment:
I'm confused as of why we are marking this PR as breaking the API. We are keeping the old call (but deprecating it), so it is still compatible, right?

Reply:
yes, will remove the `breaksAPI` tag from the PR.